ewc-rfc004-individual-wallet-attestation.md

```diff
@@ -0,0 +1,146 @@
+
+**Authors:** 
+* Mr Sebastian Bickerle (Lissi GmbH, Germany)
+* Mr Edward Curran (Lissi GmbH, Germany)
+
+**Reviewers:**
+* Dr Andreas Abraham (ValidatedID, Spain)
+* Mr George Padaytti (iGrant.io, Sweden)
+* Mr Lal Chandran (iGrant.io, Sweden)
+* Dr Nikos Triantafyllou (University of the Aegean, Greece)
+
+**Status:**
+Current: OAuth 2.0 Attestation-Based Client Authentication Draft 03 [[1](#60reference)]
+
+**Table of Contents**
+
+- [EWC RFC004: Individual Wallet Attestation - v1.0](#ewc-rfc004-individual-wallet-attestation---v10)
+- [1.0	Summary](#10summary)
+- [2.0	Motivation](#20motivation)
+- [3.0	Wallet Trust Evidence (WTE)](#30wallet-trust-evidence-wte)
+  - [3.1 Wallet Trust Evidence JWT](#31-wallet-trust-evidence-jwt)
+    - [3.1.1 Wallet Trust Evidence PoP JWT](#311-wallet-trust-evidence-pop-jwt)
+- [4.0	Messages](#40messages)
+  - [4.1 Wallet Trust Evidence within OID4VCI](#41-wallet-trust-evidence-within-oid4vci)
+    - [4.1.2 Token Request](#412-token-request)
+- [5.0	Implementers](#50implementers)
+- [6.0	Reference](#60reference)
+
+
+# 1.0	Summary
+
+This specification defines a profile for using OAuth 2.0 Attestation-Based Client Authentication to provide Wallet Instance Attestation (WIA) and Wallet Trust Evidence (WTE) as defined within the ARF 1.4 ANNEX 2 [0] within the context of EWC. 
+
+# 2.0	Motivation
+
+An EUDI Wallet Instance's life cycle begins when the individual installs an EUDI Wallet app provided by an authorized EUDI Wallet Provider. Once installed, the EUDI Wallet Provider issues a Wallet Trust Evidence (WTE) and a Wallet Instance Attestation (WIA), and the instance is now operational [Ref: ARF 1.4, Chapter 6.5.3]. 
+Using the WIA, an attestation provider can authenticate a wallet instance and ensure that it is a valid EUDI wallet that has not been revoked before issuing an attestation. A Wallet Trust Element (WTE) ensures that the Wallet Instance meets a PID Provider's or Attestation Provider's requirements, making it suitable to receive a PID or an attestation.
+Furthermore, an attestation provider must be able to determine if a wallet instance offers functional capabilities and meets the security requirements for issuance. For this purpose, the attestation provider must receive an attestation of the wallet’s capabilities and proof of non-revocation for the wallet instance.
+
+# 3.0	Wallet Trust Evidence (WTE)
+
+The attestation of a wallet's capabilities through Wallet Trust Evidence is based on the OAuth 2.0 Attestation-Based Client Authentication draft 03 [1].
+
+## 3.1 Wallet Trust Evidence JWT
+
+The following Wallet Trust Evidence (WTE) JWT must be presented when sending the token request as defined in section 6.1 of OID4VCI[2].
+
+```
+{
+        "typ": "wallet-trust-evidence+jwt",
+        "alg": "ES256",
+        "kid": "1"
+}
+.
+{
+        "iss": "https://attestation-service.com",
+        "sub": "https://wallet-solution.com",
+        "iat": 1541493724,
+        "status": {
```

Comment:
@lalc @ntsbs Should we add a section referring to the 'status' management of wallets and point it to an 'upcoming' RFC for revocation if such a one is planned?

Reply:
We could make it optional for now

Reply:
@georgepadayatti Can you please add a section and make a commit?